api/api-archive-search/archive-search-internal/src/main/java/fr/gouv/vitamui/archive/internal/server/rulesupdate/converter/RuleOperationsConverter.java

```diff
@@ -79,33 +79,36 @@ public ManagementMetadataAction convertToVitamManagementMetadataAction(
     public RuleAction convertToVitamRuleAction(VitamUiRuleAction vitamUiRuleAction) {
         final RuleAction ruleAction = VitamUIUtils.
             copyProperties(vitamUiRuleAction, new RuleAction());
-     if(vitamUiRuleAction.getStartDate() != null) {
-    LocalDateTime startDate =
-        LocalDateTime.parse(vitamUiRuleAction.getStartDate(), ArchiveSearchConsts.ISO_FRENCH_FORMATER).withHour(0)
-            .withMinute(0).withSecond(0).withNano(0);
-    ruleAction.setStartDate(ArchiveSearchConsts.ONLY_DATE_FRENCH_FORMATER.format(startDate.plusDays(1)));
-}
+        if (vitamUiRuleAction.getStartDate() != null) {
+            LocalDateTime startDate =
+                LocalDateTime.parse(vitamUiRuleAction.getStartDate(), ArchiveSearchConsts.ISO_FRENCH_FORMATER)
+                    .withHour(0)
+                    .withMinute(0).withSecond(0).withNano(0);
```

Comment:
si jamais tu repasse sur ce code : faire un getLocalDate to localdatetime permet d’éviter de créer pleins de localdatetime